cmd/juju/charmhub/data.go

```diff
@@ -9,13 +9,14 @@ import (
 	"github.com/juju/errors"
 
 	"github.com/juju/juju/api/charmhub"
+	corecharm "github.com/juju/juju/core/charm"
 )
 
-const (
+var (
 	// SeriesAll defines a platform that targets all series.
 	SeriesAll = "all"
 	// ArchAll defines a platform that targets all architectures.
-	ArchAll = "all"
+	ArchAll = corecharm.ArchAll.String()
```

Comment:
Why not work with a corecharm.Arch and manipulate it into a string(list) where needed?